tests/utils/executeQuery.js

```diff
@@ -2,7 +2,7 @@ const schema = require("../../schema");
 const { graphql } = require("graphql");
 
 async function executeQuery(query, args = {}, ctx = {}) {
-  return await graphql(schema, query, {}, ctx, args);
+    return await graphql(schema, query, {}, ctx, args); // eslint-disable-line no-return-await
```

Comment:
i think this rule is telling you to simply `return graphql` no need for the `await`?